### PR TITLE
Add note about the PVL deprecation

### PR DESCRIPTION
**Pull request by luxas**
cc @dims @radhikapc @andrewsykim @jagosan @wlan0

### CODE REVIEW

File: release-1.8/release_notes_draft.md [hunk 1]
```diff
@@ -234,6 +234,13 @@ across the system. Here's the release [scalability validation report].
   - In v1.9, the default will be `""`, which means no built-in cloud provider extension will be enabled by default.
   - If you want to use an out-of-tree cloud provider in either version, you should use `--cloud-provider=external`
   - [PR #51312](https://github.com/kubernetes/kubernetes/pull/51312) and [announcement](https://groups.google.com/forum/#!topic/kubernetes-dev/UAxwa2inbTA)
+- The `PersistentVolumeLabel` admission controller in the API Server is deprecated
+  - The replacement is running a cloud-specific out-of-tree controller-manager (often referred to as `cloud-controller-manager`) with the
+    `PersistentVolumeLabel` controller enabled. This new controller loop does exactly what this admission controller used to do.
+  - Please stop using this admission controller in your configuration files and scripts now **unless** you're dependent on the GCE and AWS
+    **in-tree** cloud providers.
+  - The `PersistentVolumeLabel` admission controller will be removed in a future Kubernetes version, when the out-of-tree versions of the
+    GCE and AWS cloud providers move to GA (compared to alpha in this release)
```
Comment (dims) [inline release-1.8/release_notes_draft.md]: `(these are marked alpha in 1.9)` will read slightly better @luxas. Thanks!

Comment (luxas) [reply release-1.8/release_notes_draft.md]: done

**Comment by luxas** [comment]
@dims updated. Can I merge this?

File: release-1.8/release_notes_draft.md [hunk 2]
```diff
@@ -234,6 +234,13 @@ across the system. Here's the release [scalability validation report].
   - In v1.9, the default will be `""`, which means no built-in cloud provider extension will be enabled by default.
   - If you want to use an out-of-tree cloud provider in either version, you should use `--cloud-provider=external`
   - [PR #51312](https://github.com/kubernetes/kubernetes/pull/51312) and [announcement](https://groups.google.com/forum/#!topic/kubernetes-dev/UAxwa2inbTA)
+- The `PersistentVolumeLabel` admission controller in the API Server is deprecated
+  - The replacement is running a cloud-specific out-of-tree controller-manager (often referred to as `cloud-controller-manager`) with the
```
Comment (andrewsykim) [inline release-1.8/release_notes_draft.md]: this can be in-tree too

**Comment by andrewsykim** [comment]
left a small comment, lgtm though!

**Comment by dims** [comment]
@luxas yes please!

File: release-1.8/release_notes_draft.md [hunk 3]
```diff
@@ -234,6 +234,13 @@ across the system. Here's the release [scalability validation report].
...
+    `PersistentVolumeLabel` controller enabled. This new controller loop does exactly what this admission controller used to do.
+  - Please stop using this admission controller in your configuration files and scripts now **unless** you're dependent on the GCE and AWS
```
Comment (radhikapc) [inline release-1.8/release_notes_draft.md]: Do not use this admission controller in the configuration files and scripts unless you are dependent on the GCE and AWS platforms.

**Comment by luxas** [comment]
@radhikapc @andrewsykim @dims Updated

**Comment by dims** [comment]
LGTM 👍

**Pull request merged**